python/triton/ops/matmul_perf_model.py

```diff
@@ -0,0 +1,133 @@
+import torch
+import triton
+import triton._C.libtriton.triton as _triton
+import heapq
+
+def get_dram_bw(backend=None, device=None):
```

Comment:
I think that this is a more generic tools that we'd want to have in `triton.testing`

Reply:
I would also rename it to `get_dram_gbps` to make it as explicit as possible